tests/test_base.py

```diff
@@ -4,3 +4,26 @@
 def test_import():
     import qutip_cupy
     assert qutip_cupy.__version__
+
+def test_conversion_cycle():
+    from qutip_cupy import CuPyDense
+    from qutip_cupy import data
```

Comment:
This module `data` doesn't belong to `qutip_cupy`!  This is `qutip.data`, it shouldn't even be publically accessible from `qutip_cupy`, let alone used from it.

Reply:
This was really messy on my part. but I changed it here and with the call to `del` in the `__init__` file  it is solved.